ada_2_ada/ada/AddConceptIds.xsl

```diff
@@ -50,7 +54,10 @@
     <xsl:template match="/">
         <xsl:choose>
             <xsl:when test="empty($schema)">
-                <xsl:message>Schema empty, please check</xsl:message>
+                <xsl:call-template name="util:logMessage">
+                    <xsl:with-param name="level" select="$logERROR"/>
```

Comment:
This seems like a similar message as the one above, should both be either WARN or ERROR?

Reply:
No this error is when there is no schema whatsoever, that is more serious than when an individual element cannot be found ín an existing schema.